aiofreepybox/api/player.py

```diff
@@ -0,0 +1,40 @@
+class Player:
+
+    def __init__(self, access):
+        self._access = access
+
+    async def get_players(self):
+        '''
+        Get players
+        '''
+        return await self._access.get('player')
+
+    async def get_player_status(self, player_id):
+        '''
+        Get player status
+        '''
+        return await self._access.get('player/{0}/api/v6/status/'.format(player_id))
```

Comment:
We should not hardcode api version.
By the way, api version is already included in the `_access` but this is api for freebox server not player. Maybe if the api version should be the same between player and server we should re use it.

Reply:
This is the player api version, it is hardcoded to v6 in the server api. While i agree with you, i don't believe a modification is required here because if (insert freebox dev name here) adds a call for some v7 player some day the new call will be '/player/playerid/api/v7/newincrediblefeature' so they will never overlap.

Reply:
I think that the way the api version is currently handled in aiofreepybox is flawed, I'm pretty sure it does not make sense to specify an api version when using the library. The calls in the library are written against a specific version of the API, if the API changes then the calls may not work anymore, so I think that it is in fact probably more meaningful to hard-code the api version in the calls themselves.

I'm not sure what versioning scheme the Freebox team is using. For instance I don't know if `get('player/123/api/v6/status')` and `get('player/123/api/v7/status')` can give different results. If that's the case, then it means that hardcoding the api version would prevent users of the library to get the new value until we update aiofreepybox. It would makes sense if we were parsing the returned value into some Python object that would need to be updated to match the new API, but since we are passing the json untouched to the library clients, maybe it is more flexible to let users inject the api version.

Another problem with hardcoding is that if we hardcode an api version that is too high, we may break the entire library for older Freeboxes. Maybe the cleanest solution would be to have one set of implementations for each API version, and use the appropriate one depending on the api version reported by the Freebox, but this is a lot of work, basically duplicating everything. I wonder how the Freebox companion app handles that.

Reply:
testing with v6 player calls with v1 to v6 returns the same result, v7 returns Version d'API inconnue: invalid_api_version, my guess is we're fine.

Reply:
API version is easy to retreive from the server. I **assume** that version are following in player too (can't be sure because the lack of communication from free devs), that said we could reuse the api version we get from server. And that's up to users to choose it but default version could be retreived from the call to http://mafreebox.free.fr/api_version

Reply:
Added api_version with 'v6' as default value, i'd rather avoid making extra calls to the freebox to get the api_version if possible. By the way do you want me to submit a patch to also bump the main class default api_version to 'v6', some other new calls won't work out of the box if default is left to 'v3'.
`class Freepybox:
    def __init__(self, app_desc=app_desc, token_file=token_file, api_version='v3', timeout=10):`

Reply:
As we don't have a good doc from the devs we must assume that v6 API still support the calls we made previously.
That would require some testing on my side for home assistant (i don't want to break the integration !)
Let me try and i will give a feedback here ASAP.

But in my mind sounds good to use latest API and more possibilites !

Reply:
@SNoof85 Have you tested the home assistant integration with the bump of the server API version to v6?

Reply:
Just tested it right now. 
Device tracker is OK
Wifi switch is OK
Bandwith sensors are OK

Everything is fine with v6 API !